mod_reforged/config/text/text.nut

```diff
@@ -0,0 +1,10 @@
+::Reforged.Text <- {
+    function getDayHourString( _seconds )
+    {
+        local days = ::Math.floor(_seconds / ::World.getTime().SecondsPerDay);
+        local hours = ::Math.floor(_seconds / ::World.getTime().SecondsPerHour);
```

Comment:
As far as I remember, the vanilla "day" doesn't have 24 hours in it. I think it has something like 16. But I may be misremembering it, do you know how many hours it has?